### ISSUE TITLE
The same number of characters is not displayed the same way on 2 lines

### ISSUE THREAD

**Issue by ericxuo** [eteran/nedit-ng#19]
Hi,

This is not easy to explain. If I could add a picture would be great.
Here is what I've wrote (to debug my macros) : 
.	input	[12:0]	e(	input	[12:0]	e),
.	input		dd(	input			dd),

You can see that the "e(" and the "dd(" are not aligned while they should. I know the wiki editor does not display correctly what I write but this is the idea.
In other words, when I set the mouse pointer on the "e" or on the "dd", nedit-ng tells me I'm on the character number 24. It is true but the letters are not correctly vertically aligned.

Could this bug come from the fonts I'm using (Courier bold,10,-1,5,75,0,0,0,0,0) ?

Regards.

Xuo.

**Comment by eteran** [comment]
If you could attach an image, that would help a lot. You can do that typically by just dragging an image file into the text box.

That being said, I have noticed that there are circumstances where Qt's fonts are unexpectedly not fixed-width. Do you experience this issue with something like "Monospace" is a size 10? Does it happen without the bold?

Thanks

**Comment by eteran** [comment]
BTW. I would recommend that you learn markdown formatting. It's easy to work with, and github supports it.

For example, if you want your two "input" lines to be in a fixed width font on this issue, you can do that!

Just add a preceeding newline, then indent each by 4 spaces and it'll be rendered as "code" and have a fixed width font :-)

**Comment by ericxuo** [comment]
Hi,

I agree that using a picture is better. 
On the first one, the cursor is after the last "m" on the 2nd line and the character number is 14.
On the second one, the cursor is after the last "i" on the 1st line and the character number is 24.
![capture d ecran_2017-12-24_18-00-13](https://user-images.githubusercontent.com/9089499/34328037-50725db6-e8d4-11e7-910b-2049c335c3ed.png)
![capture d ecran_2017-12-24_17-59-56](https://user-images.githubusercontent.com/9089499/34328038-5091ca2a-e8d4-11e7-82e9-403c982b9557.png)

The issue comes from the fonts I have selected. I've found some others that behave correctly.
The fonts with the issue are the following : 

![capture d ecran_2017-12-24_18-03-35](https://user-images.githubusercontent.com/9089499/34328055-cd454c2c-e8d4-11e7-9c27-6db1538feffa.png)


But the "bold" (in the "Primary Font" field) can't be selected from the gui. I've added it manually.

These ones work fine (even though I don't like them too much) : 

![capture d ecran_2017-12-24_18-08-15](https://user-images.githubusercontent.com/9089499/34328080-72881642-e8d5-11e7-9c5a-d8ec5a790021.png)

Not using the "bold" font seems to remove the problem but what I've noticed too, is that I can select a font from the gui, then I close the gui, then I re-open the gui to select another font but the current selected font in the gui is not the one I've chosen before.
I don't know if this is a Qt or Mageia issue. But I close the issue as I could get a workaround by selecting another font.

Regards.

Xuo.
